handel/gulpfile.js

```diff
@@ -5,7 +5,6 @@ const fs = require('fs-extra');
 const path = require('path');
 const runSeq = require('run-sequence');
 const mergeStream = require('merge-stream');
-const mocha = require('gulp-spawn-mocha');
```

Comment:
This wasn't actually getting used and was giving me console warnings, so I killed it 💣